zebra-state/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+#![doc(html_logo_url = "https://www.zfnd.org/images/zebra-icon.png")]
+#![doc(html_root_url = "https://doc.zebra.zfnd.org/zebra_storage")]
+use futures::prelude::*;
+use std::{
+    collections::HashMap,
+    error::Error,
+    future::Future,
+    pin::Pin,
+    sync::Arc,
+    task::{Context, Poll},
+};
+use tower::{buffer::Buffer, Service};
+use zebra_chain::block::{Block, BlockHeaderHash};
+
+pub enum Request {
+    // TODO(jlusby): deprecate in the future based on our validation story
+    AddBlock { block: Arc<Block> },
+    GetBlock { hash: BlockHeaderHash },
+}
+
+pub enum Response {
+    Ok,
+    Block { block: Arc<Block> },
+}
+
+pub fn in_memory() -> impl Service<
```

Comment:
IIRC at some point when this was being written there was `mod in_memory { pub fn init() -> ... }` so that the call was `in_memory::init`; did that get dropped?

Reply:
(not particularly important, just curious)